src/SciGlassCalorimeter_geo.cpp

```diff
@@ -1,3 +1,6 @@
+// SPDX-License-Identifier: LGPL-3.0-or-later
+// Copyright (C) 2022 Dmitry Kalinkin, Wouter Deconinck
+
 /*
```

Comment:
@veprbl I had to roll back and force push with the non-duplicated xml spdx headers. Because of that I couldn't cherry-pick your commit removing the license header you had included (no chain of refs). Can you re-remove the license header? Or let me know and I can do it.